Decisions.MongoDB/DeleteBulkDocumentStep.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using DecisionsFramework;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.Design.Flow.Mapping;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace Decisions.MongoDB
+{
+    [Writable]
+    public class DeleteBulkDocumentStep : BaseDeleteStep
+    {
+        private const string DOCUMENT_ID_INPUT_NAME = "Document IDs";
+        private const string PATH_ERROR = "Error";
+
+        public DeleteBulkDocumentStep() : base() { }
+        
+        public DeleteBulkDocumentStep(string serverId) : base(serverId) { }
+        public override string StepName => "Bulk Delete Documents"; 
+
+        public override DataDescription[] InputData
+        {
+            get
+            {
+                List<DataDescription> inputs = new List<DataDescription>();
+                
+                AddInputsFromServerConfig(inputs);
+                inputs.Add(new DataDescription(GetIdPropertyType(), DOCUMENT_ID_INPUT_NAME, true));
+
+                return inputs.ToArray();
+            }
+        }
+
+        public override OutcomeScenarioData[] OutcomeScenarios => new[]
+        {
+            new OutcomeScenarioData(PATH_SUCCESS),
+            new OutcomeScenarioData(PATH_ERROR)
```

Comment:
Why is this the only step with an Error Path? We should probably just be consistent.

Reply:
No idea, the delete had it before so I copied in suit